tests/testthat/_snaps/factor2string.md

```diff
@@ -5,7 +5,8 @@
     Condition
       Error in `step_factor2string()`:
       Caused by error in `prep()`:
-      ! All columns selected for the step should be factor, or ordered.
+      x All columns selected for the step should be factor, or ordered.
+      * 2 string variables found: `w`, and `x`
```

Comment:
No need for a comma here.

Reply:
fixed by using `.or` inline markup

Reply:
closed in [`d962daa` (#1235)](https://github.com/tidymodels/recipes/pull/1235/commits/d962daaa49a3c001e6b423412ff80d40970d1d86)